### PR TITLE
Introduce swatch attributes

### PR DESCRIPTION
**Pull request by IKarbowiak**
Allow creating and assigning swatch attributes. 

[Dashboard PR](https://github.com/mirumee/saleor-dashboard/pull/1301)

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [x] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by Deni-Iconis** [comment]
@IKarbowiak What's the status on this feature?

**Comment by IKarbowiak** [comment]
> @IKarbowiak What's the status on this feature?

@maarcingebala

File: CHANGELOG.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Use GraphQL IDs instead of database IDs in export - #7240 by @IKarbowiak
 - Fix draft order tax mismatch - #7226 by @IKarbowiak
   - Introduce `calculate_order_line_total` plugin method
+- Introduce swatch attributes - #7261 by @IKarbowiak
```
Comment (korycins) [inline CHANGELOG.md:75]: Shouldn't be placed in section 3.1 of changelog?

File: saleor/graphql/attribute/descriptions.py [hunk 2]
```diff
@@ -29,6 +29,10 @@ class AttributeValueDescriptions:
     SLUG = "Internal representation of a value (unique per attribute)."
     TYPE = "Type of value (used only when `value` field is set)."
     FILE = "Represents file URL and content type (if attribute value is a file)."
+    VALUE = (
```
Comment (korycins) [inline saleor/graphql/attribute/descriptions.py:32]: `VALUE` is already defined here.

**Review by korycins — APPROVED** [review]

**Review by tomaszszymanski129 — APPROVED** [review]

**Pull request merged**